src/modules/commander/commander_helper.cpp

```diff
@@ -119,7 +119,15 @@ bool is_fixed_wing(const vehicle_status_s &current_status)
 
 bool is_ground_vehicle(const vehicle_status_s &current_status)
 {
-	return (current_status.system_type == VEHICLE_TYPE_BOAT || current_status.system_type == VEHICLE_TYPE_GROUND_ROVER);
+	return (current_status.system_type == VEHICLE_TYPE_BOAT ||
+		current_status.system_type == VEHICLE_TYPE_GROUND_ROVER);
+}
+
+bool allow_center_throttle(const vehicle_status_s &current_status)
```

Comment:
Sorry for the nitpick, but can we keep the is_ground_vehicle and just check for additional vehicle types? e.g. is_uuv

Reply:
set it to 'is_uuv_vehicle'